index.html

```diff
@@ -20,10 +20,306 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <!--Content within a section are all in their own divs so that I can align them according to their personal boxes-->
+    <!--Added the a tags on the buttons by mistake, but now when I try to get rid of them, it messes up the alignment. Not sure how to fix that-->
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+    <!--Navigation Menu-->
+    <header>
+      <div class="header">
+        <div class="container">
+          <div class="navigation">
+            <h2 class="sr-only">Site Header</h2>
 
-    -->
-  </body>
-</html>
+            <!--Logo-->
+            <img
+              class="header--navigation--logo"
+              src="./images/Wensight-Logo.png"
+              height="80"
+              alt="Company Logo"
+              />
+
+              <!--Search Bar-->
+            <form class="search--bar" >
+              <input
+              name="search--products" 
+              id="search--products"
+              type="text" 
+              placeholder="Search Products">
+              <button type="submit">
+                <img 
+                src="./images/Search-icon.png"
+                alt="Search button icon"
+                height="50">
+              </button>
+            </form>
```

Comment:
You must have a `<label>` element that tells users what the `<input>` element is collecting. 

This way they know what to type into the form. 

You can hide it visually if you'd like, but it still needs to be in the HTML code. 

Also, to properly submit your form, the button needs to be a `type="submit"` or you can have an `<input type="submit" ... />`